canonical-websites/cn.ubuntu.com
# Apply the templates from newcn project

## Done
Ported the templates and images from the newcnwebsite project.

## QA
- Run `./run`
- Check the pages look good
